wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/Interactive.kt

```diff
@@ -35,13 +37,17 @@ internal constructor(
    */
   suspend fun withdraw(
     accountAddress: String,
-    assetId: IssuedAssetId,
+    assetId: StellarAssetId,
```

Comment:
We have 3 types of asset ids: stellar, issued and fiat.
Can we either add interface that only stellar and issued implements, or replace "if" when "when" (for exhaustive pattern matching)

Reply:
I can make the pattern match more exhaustive. I didn't think it was necessary because `StellarAssetId` is either `NativeAssetId` or `IssuedAssetId`, but it's better to be safe if we ever change the SEP-38 format.

Reply:
I didn't see you were using StellarAssetId.
Then it's LGTM.
Still I prefer to use `when` over `if` (in case we add more types)